File: March 28, 2026.js
function pascalRow(n) {

  let arr = [[1]];

  for (let i = 1; i < n; i++) {
    let previousRow = arr[i - 1];
    let newRow = [1];

    for (let j = 1; j < previousRow.length; j++) {
      newRow.push(previousRow[j - 1] + previousRow[j]);
      }

      newRow.push(1);
      arr.push(newRow);
    }
    
  return arr[n-1]
}

File: March 28, 2026.py
def pascal_row(n):
    arr = [[1]]

    for i in range(1, n):
        previous_row = arr[i - 1]
        new_row = [1]

        for j in range(1, len(previous_row)):
            new_row.append(previous_row[j - 1] + previous_row[j])

        new_row.append(1)
        arr.append(new_row)

    return arr[n - 1]